### PR TITLE
ARROW-12342: [Packaging] Fix tabulation in crossbow templates for submitting nightly builds

### PR DESCRIPTION
**Pull request by kszucs**
The generated templates can be checked using `archery crossbow render <task-name> --arrow-branch master`

### CODE REVIEW

**Comment by domoritz** [comment]
10K 🎉

**Comment by kszucs** [comment]
Submitted the nightly builds manually https://github.com/ursacomputing/crossbow/branches/all?query=build-206

**Comment by kszucs** [comment]
Discovered a typo in in one of the macros, so submitted the wheel tasks again  https://github.com/ursacomputing/crossbow/branches/all?query=build-206

**Comment by kszucs** [comment]
> 10K 🎉

Not a nice patch though.

File: dev/tasks/linux-packages/travis.linux.arm64.yml [hunk 1]
```diff
@@ -131,7 +131,7 @@ script:
        echo "Name-Real: Test"; \
        echo "Name-Email: test@example.com"; \
        echo "%no-protection") | \
-        gpg --full-generate-key --batch
+       gpg --full-generate-key --batch
```
Comment (kou) [inline dev/tasks/linux-packages/travis.linux.arm64.yml:134]: Could you keep this indentation?
It's for piped `gpg`.

Comment (kszucs) [reply dev/tasks/linux-packages/travis.linux.arm64.yml]: Sorry, I missed the pipe sign.

**Review by kszucs — COMMENTED** [review]
+1

**Pull request closed without merging**